baselines/ppo2/ppo2.py

```diff
@@ -364,31 +378,37 @@ def fn(x):
             mdp_gen_params = additional_params["mdp_generation_params"]
             mdp_fn = LayoutGenerator.mdp_gen_fn_from_dict(mdp_params=mdp_params, **mdp_gen_params)
             overcooked_env = OvercookedEnv(mdp=mdp_fn, **env_params)
-            agent = get_agent_from_model(model, additional_params["sim_threads"], is_joint_action=(run_type == "joint_ppo"))
+            agent = get_agent_from_model(model, additional_params["sim_threads"], is_joint_action=(run_type == "joint_ppo"), return_action_probs=False)
```

Comment:
Remove this conditional on having removed it from all other PRs (originating from overcooked_ai)